internal/campaigns/archive_fetcher.go

```diff
@@ -67,6 +68,11 @@ func unzipToTempDir(ctx context.Context, zipFile, tempDir, tempFilePrefix string
 	if err != nil {
 		return "", err
 	}
+
+	if err := os.Chmod(volumeDir, 0777); err != nil {
```

Comment:
No action required:
I think ultimately we might even want to make this configurable (not needed in the next few months, but.. maybe some day:) )
..for ssh tokens and what not

Reply:
Yeah, this definitely gets more complicated if and when we're passing more in here than just the Git repo.